response.proto

```diff
@@ -74,7 +74,7 @@ message PtDisplayInfo {
     optional hasEquipments has_equipments = 11;
     optional string name = 12;
     repeated Message messages = 13;
-    repeated Disruption disruptions = 14;
```

Comment:
Hum I think you need to change the number to handle the applicative update.

And I don't know if we need to keep the old field for the update.